compiler/rustc_session/src/config.rs

```diff
@@ -163,6 +163,17 @@ pub enum LtoCli {
     Unspecified,
 }
 
+/// The different settings that the `-Z dump_mir_spanview` flag can have.
```

Comment:
Could you explain a bit more about what each of these does?

Reply:
Let me know if this is reasonable. Since other CLI flag enums have very brief variant docs, I'm leaving those as-is, but adding to the doc for the enum as a whole:
```rust
/// The different settings that the `-Z dump_mir_spanview` flag can have. `Statement` generates a
/// document highlighting each span of every statement (including terminators). `Terminator` and
/// `Block` highlight a single span per `BasicBlock`: the span of the block's `Terminator`, or a
/// computed span for the block, representing the entire range, covering the block's terminator and
/// all of its statements.
```
OK?

Reply:
I uploaded this change